src/write-archive/archive-file.ts

```diff
@@ -30,8 +30,7 @@ export class ArchiveFile {
   }
 
   originalSrc() {
-    // Assets that we capture will be stripped of the domain in the source
-    return `${this.url.pathname}${this.url.search}`;
+    return this.url.toString();
```

Comment:
This is the main change required. We're now expecting all asset paths to be full URLs since that's what rrweb does, so we don't strip off the protocol and origin here anymore.